tuning_docs/tensile_tuning.tex

```diff
@@ -355,10 +387,13 @@ \subsubsection{BenchmarkProblems}
 
 }
 
+\textbf{BenchmarkForkParameters, JoinParameters, BenchmarkJoinParameters} are obsolete items in this section. However, they still should be in this section.
+
+
+\subsubsection{Library Logic}
 \label{sec:LibraryLogic}
-\subsubsection{LibraryLogic}
 
-The \emph{Library Logic} is a collction of one or more yaml files containing benchmarking output results (usually one per gfx architecture, per benchmark problem) created by Tensile and is the final product of the benchmarking and tuning process. It consists of kernel meta-data (called the solution), mappings to problems that it can solve and performance data at particular sizes.
+The Library Logic is a collction of one or more yaml files containing benchmarking output results (usually one per gfx architecture, per benchmark problem) created by Tensile and is the final product of the benchmarking and tuning process (dropped in 3\_LibraryLogic). It consists of kernel meta-data (called the solution), mappings to problems that it can solve and performance data at particular sizes. If we skip this section in the configuration file, the library logic file (and 3\_LibraryLogic dir) will not be generated.
```

Comment:
```suggestion
The Library Logic is a collection of one or more yaml files containing benchmarking output results (usually one per gfx architecture, per benchmark problem) created by Tensile and is the final product of the benchmarking and tuning process (dropped in 3\_LibraryLogic). It consists of a list of solutions, a list of problem sizes, and a mapping from each problem size to the fastest solution. If we skip this section in the configuration file, the library logic file (and 3\_LibraryLogic dir) will not be generated.
```

Reply:
The information here wasn't really correct imo. I think this is more accurate to what a Library Logic file actually contains.